src/archanaNJun20/SumOfDigit.java

```diff
@@ -0,0 +1,19 @@
+package archanaNJun20;
+
+public class SumOfDigit {
+	void printSumOfDigit(String input){
+		int sum=0;
+		for(int index=0;index<input.length();index++){
+			if(Character.isDigit(input.charAt(index))){
+				String charactertostring=String.valueOf(input.charAt(index));
+				int stringtointeger=Integer.parseInt(charactertostring);
+				sum+=stringtointeger;
+			}
```

Comment:
You can use below method to convert char to int
Character.getNumericValue(numberString.charAt(index));